refactor: blackbox testing (#205)
kyma-project/kyma-companion: tests/blackbox/README.md

The black box testing framework provides evaluation tests for Kyma Companion and validation of large language models (LLMs) as assessments for tests. The evaluation tests measure the improvement of Kyma Companion, while validation finds the best model to verify the test scenarios.

## Testing Data

> The data we use for both `Validation` and `Evaluation` is stored in `kyma-companion/tests/blackbox/data/namespace-scoped`. Located in that directery are a number of subdirectories named after the testing scenario they represent. In each of these you will find:

Reviewer: ```suggestion
The data we use for both `Validation` and `Evaluation` is stored in `kyma-companion/tests/blackbox/data/namespace-scoped`. Located in that directory are several subdirectories named after the testing scenario they represent. In each of these you will find:
```

> - `deploy.sh`, a shell script that would help to deploy the scenario on a Kubernetes cluster.

Reviewer: ```suggestion
- `deploy.sh`, a shell script that helps deploy the scenario in a Kubernetes cluster.
```

> - `deployment.yml`, a Kubernetes manifest, that is utilised by the aforementioned `deploy.sh`.

Reviewer: ```suggestion
- `deployment.yml`, a Kubernetes manifest used by the `deploy.sh` shell script.
```
